addons/repos/main.yaml

```diff
@@ -1,5 +1,8 @@
 ---
 packages:
+  - name: fluxcd-source-controller
+    versions:
+      - 0.21.2
   - name: cert-manager
     versions:
       - 1.3.3
```

Comment:
@EstebanFS Actually, don't worry about this, since FluxCD is slated for v0.11.0, let's keep this change here so that when it's merged it will be in main and ready for the release.